core/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+use syn::*;
+
+pub mod meta;
+
+pub fn extract_from_mod(input: &ItemMod) -> Vec<meta::Struct> {
+    input
+        .content
+        .as_ref()
+        .unwrap()
+        .1
+        .iter()
```

Comment:
suggestion (nb): I'd imagine that you want to collect the struct def too and store it alongside the methods (especially as we start having attributes on struct defs)